frida_mode/Makefile

```diff
@@ -1,13 +1,17 @@
 PWD:=$(shell pwd)/
+PARENT:=$(shell realpath $(PWD)..)/
 INC_DIR:=$(PWD)include/
 SRC_DIR:=$(PWD)src/
 INCLUDES:=$(wildcard $(INC_DIR)*.h)
-SOURCES:=$(wildcard $(SRC_DIR)*.c)
+SOURCES:=$(wildcard $(SRC_DIR)**/*.c) $(wildcard $(SRC_DIR)*.c)
 BUILD_DIR:=$(PWD)build/
 CFLAGS+=-fPIC -D_GNU_SOURCE
+QEMU_INC_DIR:=$(PARENT)qemu_mode/qemuafl/qemuafl/
+QEMU_INC_API:=$(QEMU_INC_DIR)api.h
```

Comment:
I'm hoping to share much of the code for qemu asan down the line. There enough similarity that it doesn't make sense to re-implement it all. So I figured a dependency on QEMU mode was preferable to duplication?

Reply:
The thing is that qemuafl is not necessarily checked out. I think it is fine if you copy the file and put it into the frida_mode include directory

Reply:
Turns out it's only a single structure I need for the CPU regs, so I have just duplicated it. I only need it as I have reused the `utils/aflpp_driver/aflpp_qemu_driver_hook.so` and so need to pass the context in the same format.

Down the line, its [this](https://github.com/AFLplusplus/AFLplusplus/tree/stable/qemu_mode/libqasan) code I expect to be using, so that's not in qemuafl and so doesn't have the same problems.